optimas/diagnostics/exploration_diagnostics.py

```diff
@@ -851,3 +858,83 @@ def _check_pareto_objectives(
                 if isinstance(objective, str):
                     objectives[i] = self._get_objective(objective)
         return objectives
+
+    def print_evaluation(self, trial_index: int) -> None:
+        """Print the parameters of the given evaluation.
+
+        Parameters
+        ----------
+        trial_index : int
+            Index of an evaluated trial.
+        """
+        h = self.history.loc[trial_index]
+        print("Evaluation %i: " % (trial_index))
+        print("%20s = %s" % ("sim_id", h["sim_id"]))
+
+        try:
+            sim_path = self.get_evaluation_dir_path(trial_index)
+            print(
+                "%20s = %s"
+                % ("dir_path", self.get_evaluation_dir_path(trial_index))
+            )
+        except ValueError:
+            sim_path = None
+            print("%20s = None" % ("dir_path"))
+
+        print("objective functions:")
+        objective_names = [obj.name for obj in self.objectives]
+        for name in objective_names:
+            print("%20s = %10.5f" % (name, h[name]))
+
+        print("varying parameters:")
+        varpar_names = [var.name for var in self.varying_parameters]
+        for name in varpar_names:
+            print("%20s = %10.5f" % (name, h[name]))
+
+        if len(self.analyzed_parameters) > 0:
+            anapar_names = [var.name for var in self.analyzed_parameters]
+            print("analyzed parameters:")
+            for name in anapar_names:
+                print("%20s = %10.5f" % (name, h[name]))
+
+        print()
+
+    def print_top_evaluations(
```

Comment:
To keek it consistent with the other methods, I would change the name to
```suggestion
    def print_best_evaluations(
```

Reply:
alright!